packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -84,6 +86,22 @@ class TooltipThemeData with Diagnosticable {
   /// The length of time that the tooltip will be shown once it has appeared.
   final Duration? showDuration;
 
+  /// The [TooltipTriggerMode] that will show the tooltip.
+  ///
+  /// This value is used if [Tooltip.triggerMode] is null.
+  /// Read [Tooltip.triggerMode] documentation for other sources.
```

Comment:
This is not very helpful. It should be structured as a `See also` with more context or just put the information you mean to convey here. If there is documentation duplication, you can template it to reuse in more than one place. Search for `{@macro...` and `{@template...` to see how to reuse docs.